test/run_tests/smoke_test/smoketest.sh

```diff
@@ -5,8 +5,8 @@
 
 path=$(dirname "$0")
 
-# assuming SW_APM_SERVICE_KEY is set
-export SW_APM_COLLECTOR=collector-stg.appoptics.com
+# use staging, assuming SW_APM_SERVICE_KEY is set
+export SW_APM_COLLECTOR=apm.collector.st-ssp.solarwinds.com
```

Comment:
Is this going to be the "forever" url of staging?

Reply:
I don't think so, but the one that will most likely work in the short term